is_nan.json

```diff
@@ -1,9 +1,10 @@
 {
     "id": "is_nan",
     "summary": "Value is not a number",
-    "description": "Checks whether the specified value `x` is not a number (often abbreviated as `NaN`). The definition of `NaN` follows the [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935). All non-numeric data types MUST also return `true`.",
+    "description": "Checks whether the specified value `x` is not a number (often abbreviated as `NaN`). The definition of `NaN` follows the [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935). All non-numeric data types MUST also return `true`, including arrays that contain `NaN` values.",
     "categories": [
```

Comment:
> is not a number (often abbreviated as `NaN`)

this is a bit misleading because there is a difference between "not a number" (object is not an int or float) and "NaN" (which is a special IEEE float value).

Reply:
proposal

> Checks whether the specified value `x` is not a number. Returns `True` for numeric values (integers and floats), except for the special value `NaN` as defined by the [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935). All non-numeric data types MUST also return `true`, including arrays that contain `NaN` values.

Reply:
Sounds good.